tendermint/src/node/id.rs

```diff
@@ -104,7 +104,7 @@ impl TryFrom<PublicKey> for Id {
     fn try_from(pk: PublicKey) -> Result<Self, Self::Error> {
         match pk {
             PublicKey::Ed25519(ed25519) => Ok(Id::from(ed25519)),
-            _ => Err(Kind::UnsupportedKeyType.into()),
+            // _ => Err(Kind::UnsupportedKeyType.into()),
```

Comment:
this was giving me unreachable code error

Reply:
This happens when compiled without all features enabled, otherwise it should be correct when the tendermint crate is compiled with its features as one of the variants is guarded:

https://github.com/informalsystems/tendermint-rs/blob/2609b60755114c73ab6b5d4b169463e1de95b243/tendermint/src/public_key.rs#L49-L57

Reply:
I would expect this to be specified in the readme or `Makefile`. @thanethomson should I update the readme or create a Makefile target?